hathor/transaction/storage/transaction_storage.py

```diff
@@ -1195,7 +1230,7 @@ def _run_topological_sort_dfs(self, root: BaseTransaction, visited: Dict[bytes,
             for txin in tx.inputs:
                 if txin.tx_id not in visited:
                     try:
-                        txinput = self.get_transaction(txin.tx_id)
+                        txinput = self.get_transaction(txin.tx_id, allow_partially_valid=True)
```

Comment:
Should we add a parameter to the method?

Reply:
Maybe. I tested just removing it and it breaks at least these tests:

```
FAILED tests/others/test_init_manager.py::SyncV1ManagerInitializationTestCase::test_init_good_order - SystemExit
FAILED tests/others/test_init_manager.py::SyncBridgeManagerInitializationTestCase::test_init_good_order - SystemExit
FAILED tests/others/test_init_manager.py::SyncV2ManagerInitializationTestCase::test_init_good_order - SystemExit
```